packages/mobile/src/identity/reducer.ts

```diff
@@ -309,7 +317,7 @@ export const reducer = (
         feelessVerificationStatus: action.status,
         feelessVerificationState: {
           ...state.feelessVerificationState,
-          isActive: action.status > 0 && action.status !== VerificationStatus.Done,
```

Comment:
Why is the `> 0` not needed anymore? Are those states simply not possible anymore? Even if they're not I don't see an issue with leaving it just in case, but I might be missing something.

Reply:
The issue was that we were using `feelessVerificationState.isActive` as a way to know whether to check for the feeless verification error or the non-feeless verification error. When we had `status > 0` that would set `isActive = false` whenever there was an error, which would cause the UI to check for non-feeless errors.

This is messy logic and mostly just a bandage that will hopefully be simplified with the verification refactor work.